tests/compatibility-test.py

```diff
@@ -8,7 +8,14 @@
 import docker
 
 from kuberay_utils import utils
-from kubernetes import client, config
+from framework.prototype import (
+    CONST,
```

Comment:
All of the objects that are not directly related to the prototype test framework should probably live in the utils file.

The objects that should be moved include `CONST`, `K8S_CLUSTER_MANAGER`, `OperatorManager`, and `shell_subprocess_run`.

Reply:
Is this necessary in this PR? I am not familiar with `import` in Python. If we need to move them into a separate module, I need time to understand the import mechanism of Python. If it is not required in this PR, I can open an issue to track this issue.

Reply:
It's not so important for this PR and can be done later!
Utilities used by all tests (like `shell_subprocess_run`) should probably be collected in a "utilities" file.

Reply:
Open an issue #774 to track the progress.